components/Registration/Registration.tsx

```diff
@@ -64,10 +104,20 @@ const RegistrationForm: React.FC<RegistrationFormProps> = ({
             setFurthestPage(newIndex);
         }
         window.scroll(0, 0); // Scroll to top of page
+        contentRef?.current?.scroll({
+            // Necessary for mobile: the content div is fixed to the screen height, and is scrollable
+            top: 0,
+            left: 0
+        });
     };
 
     const previousPage = () => {
         console.log("prev");
+        window.scrollTo(0, 0);
```

Comment:
doesn't this function call handlePageChange which handles scrolling

Reply:
I initially added the `contentRef` to hold the mobile-specific backgrounds in the same place, but removed it as it breaks the Navbar functionality. window.scrollTo(0,0) now handles all scrolling back to the top of the page.